Cargo.toml

```diff
@@ -21,8 +21,7 @@ default = ["std"]
 std = ["alloc", "secp256k1-sys/std"]
 # allow use of Secp256k1::new and related API that requires an allocator
 alloc = ["secp256k1-sys/alloc"]
-bitcoin-hashes = ["bitcoin_hashes"] # Feature alias because of the underscore.
-bitcoin-hashes-std = ["std", "bitcoin_hashes", "bitcoin_hashes/std"]
+hashes-std = ["std", "hashes/std"]
```

Comment:
Whenever we hit MSRV 1.60 we can replace all the `-std` ones with:
`std = ["hashes?/std"]`

Reply:
Already tracked here: https://github.com/rust-bitcoin/rust-bitcoin/discussions/1722 but thanks for mentioning, I didn't actually know what "weak dependencies" meant - TIL.